test/autorest/generated/complexgroup/internal/complexgroup/service.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package complexgroup
+
+import (
+	"net/http"
+	"net/url"
+	"path"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+// Service ..
+type Service struct{}
+
+// GetValidCreateRequest creates the GetValid request.
+func (Service) GetValidCreateRequest(u url.URL) (*azcore.Request, error) {
+	u.Path = path.Join(u.Path, "/complex/basic/valid")
```

Comment:
The fact that this doesn't need an `api-version` query param is a bug in the [response file](https://github.com/Azure/autorest.testserver/blob/master/mappings/vanilla/complex_basic_valid-62466d5b-943d-47d3-a192-520d96d5d6c3.json#L5).  Can you please confirm?

Reply:
Yes, GetValid is currently able to respond without the api-version query parameter. PutValid on the other hand requires api-version in order to accept the request.